spec/feature/authenticated_user_spec.rb

```diff
@@ -205,79 +241,16 @@
     click_link_or_button "Reservation 00001"
     expect(page).to have_content("Reservation 00001")
   end
-  #
-  # it "can edit their profile on their own page" do
-  #   allow_any_instance_of(ApplicationController).to receive(:current_user).
-  #                                                   and_return(valid_user)
-  #   visit user_path(valid_user)
-  #   expect(current_path).to eq(user_path(valid_user))
-  #   click_link_or_button('Edit Profile')
-  #   fill_in "user[about_me]", with: "I'm no fun!"
-  #   click_link_or_button 'Submit'
-  #   expect(page).to have_content("I'm no fun!")
-  # end
-  #
-  # context "can view the reservation page with" do
-  #   before(:each) do
-  #     allow_any_instance_of(ApplicationController).to receive(:current_user).
-  #     and_return(valid_user)
-  #     click_add_to_cart_link("Breakfast")
-  #     click_link_or_button "Cart:"
-  #     click_link_or_button "Checkout"
-  #   end
-  #
-  #   it "items with quantity ordered and line-item subtotals" do
-  #     within("#item-quantity") do
-  #       expect(page).to have_content("1")
-  #     end
-  #     within("#item-subtotal") do
-  #       expect(page).to have_content("$10.00")
-  #     end
-  #   end
-  #
-  #   it "items with links to each item description page" do
-  #     within("#item-title") do
-  #       click_link_or_button "Bacon"
-  #     end
-  #     expect(page).to have_content("Bacon")
-  #   end
-  #
-  #   it "the current status of the reservation" do
-  #     within("#reservation-status") do
-  #       expect(page).to have_content("ordered")
-  #     end
-  #   end
-  #
-  #   it "reservation total price" do
-  #     within("#item-total") do
-  #       expect(page).to have_content("$10.00")
-  #     end
-  #   end
-  #
-  #   xit "date/time reservation was submitted" do
-  #     within("#reservation-submit-time") do
-  #       expect(page).to have_content("Reservation Submitted At:")
-  #     end
-  #   end
-  # end
-
-  # context ", when an item is retired," do
-  #   before(:each) do
-  #     allow_any_instance_of(ApplicationController).to receive(:current_user).
-  #                                                     and_return(valid_user)
-  #     click_add_to_cart_link("Breakfast")
-  #     item = Item.find_by(title: "Bacon and Eggs")
-  #     item.update(status: "hidden")
-  #     click_link_or_button "Cart:"
-  #     click_link_or_button "Checkout"
-  #   end
-  #
-  #   xit "can still access the item page" do
-  #     click_link_or_button "Bacon and Eggs"
-  #     expect(current_path).to eq(item_path(1))
-  #     expect(page).to have_content("Bacon and Eggs")
-  #   end
-  # end
+
+  it "can edit their profile on their own page" do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+                                                    and_return(valid_user)
+    visit user_path(valid_user)
+    click_link_or_button('Edit Profile')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.